package.json

```diff
@@ -38,6 +38,7 @@
     "clang-format": "^1.4.0",
     "cli-progress": "^3.7.0",
     "conventional-commits-parser": "^3.2.1",
+    "del": "^6.0.0",
```

Comment:
I think that its probably not worth it since releases will (nearly) always run on unix systems where the file removal is pretty quick.  And not that its the biggest deal in the world in terms of size, but its yet another dependency we are carrying, with minimal value gained.

Reply:
that's fair. will update